### PR TITLE
search: clarify that the case of the `name` is as requested or all lower

### PR DESCRIPTION
**Pull request by dcbaker**
This means that if the requester (whether that is directly from a user or from another CPS file) specifies the name in a case that is not all lower, search for that exactly as well as the all lower.

For example, if "OpenGL" is requested, then the name should be searched as "OpenGL" and as "opengl".

I struggled a little bit with what to call the "requester", which I still don't like but can't come up with anything better.

### CODE REVIEW

File: searching.rst
Comment (mwoehlke) [inline searching.rst:56]: Two minor points: first, I think "consumer" would be more accurate here¹, and second, ["try to break at 'natural' boundaries"](https://cps-org.github.io/cps/development.html#contributing).

```suggestion
  including both the case as specified by the consumer,
  and the name converted to lower case.
```

\- or -

```suggestion
  The name of the package to be located, including both
  the case as specified by the consumer,
  and the name converted to lower case.
```

Either is fine; use whichever you think expresses the more natural "thought break". Note that the second needs to also alter the preceding line, and can't be (correctly) applied via github. (FWIW, I'm slightly inclined to the latter.)

"user / consumer" would also be acceptable.

¹ We do use "user" in a few places. A glossary might be helpful; I *think* we're consistently using "user" to refer to the ultimate consumer, especially e.g. the point where a human has written `find_package` or invoked `cps-config`. A "consumer" includes transitive consumers, e.g. when a package depends on another package. Thus, I think "consumer" is superior here.

**Review by mwoehlke — COMMENTED** [review]
See nit. Thanks for the PR!

**Review by mwoehlke — APPROVED** [review]

**Comment by mwoehlke** [comment]
Sigh, I give up waiting on a second approval. Merging.

**Comment by dcbaker** [comment]
Maybe it makes sense for clarifications and such only require one approval vs material changes that require more than one?

**Pull request merged**
